tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -3467,7 +3476,6 @@ SpirvVariable *DeclResultIdMapper::getBuiltinVar(spv::BuiltIn builtIn,
     return builtInVar->second;
   }
   bool mayNeedFlatDecoration = false;
-  spv::StorageClass sc = spv::StorageClass::Max;
   // Valid builtins supported
   switch (builtIn) {
```

Comment:
I would move this code to the version of the `getBuiltinVar` that does not take a storage class.

Reply:
Done.